src/index.ts

```diff
@@ -19,21 +19,42 @@ async function run(): Promise<void> {
     }
 
     const octokit = new Octokit(options);
-    const pullRequest = await octokit.pulls.create(pullParams);
-    const pullNumber = pullRequest.data.number;
-    const htmlUrl = pullRequest.data.html_url;
-
-    if (reviewers.length > 0) {
-      await octokit.pulls.createReviewRequest({
-        owner: pullParams.owner,
-        repo: pullParams.repo,
-        pull_number: pullNumber,
-        reviewers,
-      });
-    }
+    try {
```

Comment:
Do we have to do nested `try`s? I think there should be a way of having only a single one.

Reply:
fixed